action.yml

```diff
@@ -1,4 +1,4 @@
-name: "Select Star dbt impact report"
+name: "Select Star dbt Impact Report Action"
```

Comment:
`Select Star dbt Impact Report` I don't think we decided to put "Action" in the name, 

https://selectstarhq.slack.com/archives/C05JEHLNL4U/p1696974971154139?thread_ts=1696958175.739519&cid=C05JEHLNL4U